x-pack/test/observability_functional/apps/observability/alerts/bulk_actions.ts

```diff
@@ -1,204 +0,0 @@
-
-  const retry = getService('retry');
-
-  // Hide the WorkFlow filter, but keep its code as required in https://github.com/elastic/kibana/issues/117686
-  describe.skip('Observability alerts / Bulk actions', function () {
-    this.tags('includeFirefox');
-    before(async () => {
-      await esArchiver.load('x-pack/test/functional/es_archives/observability/alerts');
-      await esArchiver.load('x-pack/test/functional/es_archives/infra/metrics_and_logs');
-      await esArchiver.load(
-        'x-pack/test/apm_api_integration/common/fixtures/es_archiver/apm_8.0.0'
-      );
-    });
-
-    after(async () => {
-      await esArchiver.unload('x-pack/test/functional/es_archives/infra/metrics_and_logs');
-      await esArchiver.unload('x-pack/test/functional/es_archives/observability/alerts');
-      await esArchiver.unload(
-        'x-pack/test/apm_api_integration/common/fixtures/es_archiver/apm_8.0.0'
-      );
-    });
-
-    describe('When user has all priviledges for logs app', () => {
-      before(async () => {
-        await observability.users.setTestUserRole(
-          observability.users.defineBasicObservabilityRole({
-            logs: ['all'],
-          })
-        );
-        await observability.alerts.common.navigateToTimeWithData();
-      });
-
-      after(async () => {
-        await observability.users.restoreDefaultTestUserRole();
-      });
-
-      it('logs checkboxes are enabled', async () => {
-        const logsCheckboxes =
-          await observability.alerts.bulkActions.getCheckboxSelectorPerProducer('logs');
-
-        await asyncForEach(logsCheckboxes, async (checkbox, index) => {
-          expect(await checkbox.isEnabled()).to.be(true);
-        });
-      });
-
-      describe('when checkbox is clicked', async () => {
-        it('shows bulk actions container', async () => {
-          const logsCheckboxes =
-            await observability.alerts.bulkActions.getCheckboxSelectorPerProducer('logs');
-          await logsCheckboxes[0].click();
-          await observability.alerts.bulkActions.getBulkActionsContainerOrFail();
-        });
-
-        describe('when selected bulk action button is clicked', async () => {
-          it('opens overflow menu with workflow status options', async () => {
-            await retry.try(async () => {
-              await (await observability.alerts.bulkActions.getBulkActionsButton()).click();
-            });
-          });
-        });
-      });
-    });
-
-    describe('When user has all priviledges for apm app', () => {
-      before(async () => {
-        await observability.users.setTestUserRole(
-          observability.users.defineBasicObservabilityRole({
-            apm: ['all'],
-          })
-        );
-        await observability.alerts.common.navigateToTimeWithData();
-      });
-
-      after(async () => {
-        await observability.users.restoreDefaultTestUserRole();
-      });
-
-      it('apm checkboxes are enabled', async () => {
```

Comment:
@fkanout Same here, I would change the expectation to "apm checkboxes are not visible"

Reply:
same as here https://github.com/elastic/kibana/pull/120231#discussion_r764740220